lib/engine/game/base.rb

```diff
@@ -184,7 +184,7 @@ def self.load_from_json(json)
         const_set(:HEXES, data['hexes'])
       end
 
-      def initialize(names, id: 0, actions: [])
+      def initialize(names, id: 0, actions: [], pin: nil)
```

Comment:
should we call all the other references pin as well? as opposed to pin_version

Reply:
Can do, will change them all to pin